lib/common/acl.c

```diff
@@ -276,6 +277,34 @@ pcmk__apply_acl(xmlNode *xml)
     }
 }
 
+static bool
+pcmk__is_user_in_group(const char *user, const char *group)
```

Comment:
Let's put this in lib/common/utils.c so we can create a unit test for it. It can be declared in lib/common/crmcommon_private.h since it will only be used in the library.